g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/crons/SourceSendGRMCron.java

```diff
@@ -19,15 +19,15 @@
 @Service
 @ConditionalOnProperty(value="sync.source.enable", havingValue="true",  matchIfMissing=false)
 public class SourceSendGRMCron {
-	
+	private final static long THREESECOND = 3000;
 	@Autowired
 	private SyncGrmSourceService syncGrmSourceService;
 	
 	
 	/**
 	 * Synchronize the updated source to remote GRM server
 	 */
-	@Scheduled(cron = "${sync.source.schedule.cron}")
+	@Scheduled(fixedDelay = THREESECOND)
```

Comment:
why here it's @Scheduled(fixedDelay = THREESECOND), but in 'g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/crons/SourceLocalBundleCron.java' it's @Scheduled(cron = "${sync.source.schedule.cron}")?

Reply:
the tester requested that send to GRM source immediately after source merge to local bundle. there configure scanning the GRM source directory  every three second